### DISCUSSION TITLE
吃不满带宽

### DISCUSSION THREAD

**Discussion by ho1yman** [localsend/localsend#400] [L-简体中文]
刚传一个2.4GB的mp4文件，传输速率只有40兆字节每秒。路由器是ax86u，网线的话发端和收端都是插的cat8的网线，所以正常应该是100兆字节每秒。
![image](https://user-images.githubusercontent.com/93108144/233017051-d7b8f0f3-75b8-4ab8-89f6-f2dec547f0c2.png)

**Comment by logaway** [comment]
会不会是QOS限制，或是手机WIFI的限制？

**Comment by jp1017** [comment]
我的怎么最大才8MB/s呢，怎么回事？电脑建热点，手机连电脑热点也是8呢，需要调节哪里？

**Reply by tiny-andr** [reply]
牛逼牛逼。关闭加速前只有12MB每秒，关闭之后直接50多

**Comment by jp1017** [comment]
> 可以试试在设置里把加密关了。我一开始也是速度只有8兆，关了加密速度就起飞了。

加密关闭，也不行，快了点点，还达不到飞速呢，咋回事？？

**Reply by blngxj** [reply]
果然关了加密快很多。但是在局域网内加密似乎不是硬需求，希望开发者默认切至关闭，给有需求朋友加一条提示

**Comment by jp1017** [comment]
> 刚传一个2.4GB的mp4文件，传输速率只有40兆字节每秒。路由器是ax86u，网线的话发端和收端都是插的cat8的网线，所以正常应该是100兆字节每秒。

您好，可以截图发下您的设置吗，加密关了吗？

**Comment by ikidou** [comment]
同样的问题，手机上跑SFTP的话内网基本上可以到50-60MB/s 左右，但是刚刚发现只要开了加密就只有8MB/s，关了就有50-60MB/s，按说都是加密，为啥这个慢这么多，加密算法太复杂了吗

**Reply by KingDoum** [reply]
不知道为啥，我开了加密

电脑传输给平板   速度20MB  关了加密 速度是30MB每秒
平板传输给电脑 关了加密 40MB每秒，难道是路由器问题，限制了么

**Comment by touryung** [comment]
我的更牛，关了加密还是2M/s，我这还千兆宽带呢，不知道怎么回事

**Reply by RYF520** [reply]
这走的是局域网，和宽带没关系，和路由器有关，但是你这个两兆也确实有点慢

**Reply by LogicalNotZero** [reply]
是不是离路由器太远了点，一般传大文件的时候，手机就丢在路由器旁边。

**Comment by ikidou** [comment]
好像说是Dart的Https有问题

**Comment by tom8zds** [comment]
dart 的问题, 我用rust写了一个, 能跑500M
![image](https://github.com/localsend/localsend/assets/20534594/b3794851-5ef3-414e-99f6-5dca32970f62)

**Reply by yexua** [reply]
老哥开源了吗？

**Comment by MoChen2749** [comment]
我把加密关了还是跑不满，双电脑互传（ThinkPad 450s和redmiG，路由器是红米AC2100AP模式+iStoreOS千兆）

**Comment by YBxbxxz** [comment]
所以有啥其他可代替的软件吗,这个确实比较慢

**Comment by ho1yman** [comment]
用nginx呗，我用了好多局域网传文件的方式，感觉只有nginx可以拉满带宽。。

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ZimaKlein** [comment]
我连手机热点也是40MB/s，这是上限了吗？

**Comment by Keeevink** [comment]
我这边手机和电脑都关闭了加密，可以稳定在117左右
![image](https://github.com/localsend/localsend/assets/54541925/c3744e44-02ad-4c0b-b649-e58c37306232)

**Comment by Titans11** [comment]
关闭加密是正解！ 但是别忘记了：客户端网络里设置“重启”才能生效，手机电脑都样，必须重启！局域网内非加密状态下，跑满带宽！爽到飞起！

**Comment by wwwwww1239** [comment]
没关加密2M/s,关闭加密后8M/s,正常wifi100M/s正常吗？不过快了4倍不传大东西也能用了......

**Reply by wwwwww1239** [reply]
不对，现在电脑和手机之间有80M/s了！！

**Reply by Titans11** [reply]
恭喜，加油，好用，免费，无敌

**Comment by amdfanvip** [comment]
我的win 到mac20-24MB/s
mac到win是 33.6MB/s
win的无线连接是2402Mb
mac的无线连接是780Mb
同一路由下

**Comment by pricky777** [comment]
同 AX86U，之前一直也是 40MB/s 左右，看了帖子刚才关掉加密，直接跑到 145MB/s 了。。。

**Comment by Morellono** [comment]
两台电脑都关了加密，一台发送时能跑到180M，但另一台跟加密前没变化，十几M。用手机测试同样，只有一台电脑发送速度快，手机接收没变化

**Comment by xihale** [comment]
基于tls的加密很慢，如果是家庭网络并且与外界阻隔其实大可以放心关闭加密。
其实如果换用tcp&（使用https传输的）sha512验证感觉会好一些。

**Comment by youtonghy** [comment]
我也遇到了这个问题，电脑和iPhone和iPad互传数据可以达到700M左右的速度，但是小米和华为都只能跑200M，很奇怪
